documentation/List-of-Action-plugins.md

```diff
@@ -658,6 +659,26 @@ Can supply *either* `applicationId` or `applicationSerial`, although this can be
 
 Note: There is a database trigger/postgres listener to automatically delete files when their database record is deleted, so we only need to delete the records, not the files themselves.
 
+### Aliasing existing template actions
+
+Finally, we have this "special" action which allows us to create an alias to an existing template action. The use for this is if you have configured a complex action with a lot of parameters or queries, and you wish to run it with slight differences for a document preview (for example), you can just set an alias to the original action instead of configuring and maintaining the same thing twice within the template (LINK TO PREVIEW DOCUMENTATION WHEN ITS DONE)
+
+In fact, this is not really an action at all -- it's just a dummy plugin that makes the action library and front-end (and therefore template_actions) see it as an available action. But all the logic is handled internally as part of the `processTrigger` functionality -- if it sees an "alias" action, the function simply swaps it out with the template action it is referencing.
+
+- _Action Code:_ **`alias`**
+
+| Input parameters<br />(\*required) <br/> | Output properties                  |
+| ---------------------------------------- | ---------------------------------- |
+| `code`\*                                 | `output as per the aliased action` |
+| `condition`                              |                                    |
+| `...any other aliased action parameters` |                                    |
+
+To be able to reference an existing template action, it needs to have a unique (per template) code in its template_action record.
+
+Then this alias action just passes in the code of the action it's referencing and the trigger processor will swap it out for that one.
+
+By default the action runs with the condition and parameters it has defined, but these can be overridden by the alias. To override the condition, you can either set the "condition" field directly, or supply a "condition" parameter (which takes priority). The one problem with the native "condition" field, though, is that it is always `true` by default, so we wouldn't want to always override with that. Therefore, the "condition" field only overrides the aliased actions condition if it is something other than `true`. (If you want it to be `true` (i.e. always run, then set `condition: true` in parameters))
```

Comment:
This is not 100% clear, an example would be great. 

>  Therefore, the "condition" field only overrides the aliased actions condition if it is something other than `true`. (If you want it to be `true` (i.e. always run, then set `condition: true` in parameters))

Like this one, to me it read, `condition overrides aliased condition if value other then true, to alway run set condition: true` (which seems to contradict)

Reply:
Ohh i think my confusion is between `field` and `parameter`, maybe we can differentiate somehow

Reply:
See below i've have a suggestion